pkg/pb/gracefulexit.proto

```diff
@@ -10,8 +10,44 @@ import "orders.proto";
 
 package gracefulexit;
 
+// NodeGracefulExit is a private service on storagenodes
+service NodeGracefulExit {
+  // GetSatellitesList returns a list of satellites that the storagenode has not begun a graceful exit for.
+  rpc GetNonExitingSatellites(GetNonExitingSatellitesRequest) returns (GetNonExitingSatellitesResponse);
+  // StartExit updates one or more satellites in the storagenode's database to be gracefully exiting.
+  rpc StartExit(StartExitRequest) returns (StartExitResponse);
+ 
+}
+
+message GetNonExitingSatellitesRequest{}
+
+message StartExitRequest {
+  repeated bytes node_ids = 1 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];
```

Comment:
Why does this have a list of satellites rather than a single satellite? This will end-up making the error handling much more confusing. A single exit request would be clearer.

Reply:
We thought it would be nice that user don't have to rerun the command for graceful exit against each satellite. I can change it back to only take a single satellite address as the input if that's a better design

Reply:
From the CLI side you could issue several requests, if needed.